src/doom/d_main.c

```diff
@@ -1673,6 +1673,18 @@ void D_DoomMain (void)
         DEH_AddStringReplacement("M_SCRNSZ", "M_DISP");
     }
 
+    //!
+    // @category game
+    // @category mod
+    //
+    // Automatic pistol start when advancing from one level to the next. At the
+    // beginning of each level, the player's health is reset to 100, their
+    // armor to 0 and their inventory is reduced to the following: pistol,
+    // fists and 50 bullets. This option has no effect when recording a demo,
+    // playing back a demo or when starting a network game.
+    //
+    crispy->pistolstart = M_ParmExists("-pistolstart");
```

Comment:
I think I'd prefer a clean line between this and the comment.

Reply:
Noted!